phpBB/includes/acp/acp_bbcodes.php

```diff
@@ -269,6 +269,7 @@ function main($id, $mode)
 
```

Comment:
Why are you not using your BBCodeMonkey in this class?
AFAIK, it is a superset of phpBB's BBCode definition for custom BBCode...